core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalSystem.java

```diff
@@ -448,6 +448,7 @@ public synchronized void gainPrimacy() {
 
     Preconditions.checkState(mRaftJournalWriter == null);
     mRaftJournalWriter = new RaftJournalWriter(nextSN, client);
+    // TODO(jiacheng): pass sth here?
```

Comment:
@ggezer how many `AsyncJournalWriter` threads are there for embedded journal? Is the `LocalFirstRaftClient` a suitable distinguisher?

Reply:
There's only one async-writer. So should be 1 only.